cmd/api/notifications/notifications_test.go

```diff
@@ -0,0 +1,84 @@
+package notifications
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/matryer/is"
+)
+
+var ntfy *Ntfy
+
+func TestMain(m *testing.M) {
+	notificationsBaseURL := "https://ntfy.sh/test_Ah3mn6oD"
+	enableNotifications := true
+	ntfy = NewNtfy(enableNotifications, notificationsBaseURL)
+
+	os.Exit(m.Run())
+}
+
+func TestBookCreated(t *testing.T) {
+
+	t.Run("notificates the criation of a new book without errors", func(t *testing.T) {
```

Comment:
Its good to have a test that hits the actual service we are using, but please add a test that uses a mocked `http.Client` to assert the http request.
You could use a round tripper to assert the requests like this (https://www.reddit.com/r/golang/comments/gp53pu/using_roundtripper_to_mock_http_requests_in_go) or define a `Doer` interface, which would model the `client.Do` method (https://stackoverflow.com/questions/43240970/how-to-mock-http-client-do-method).